docker-compose.yml

```
@@ -262,9 +262,48 @@ services:
     ports:
       - "127.0.0.1:9090:9090"
     volumes:
-      - ./prometheus.yml:/etc/prometheus/prometheus.yml
+      - ./monitoring/prometheus.yml:/etc/prometheus/prometheus.yml
       - prometheus-data:/prometheus
 
+  pyroscope:
+    profiles:
+      - monitoring
+    image: docker.io/grafana/pyroscope
```

The version tag is missing.

```suggestion
    image: docker.io/grafana/pyroscope:1.13.2
```